pkg/services/authz/zanzana/proto/v1/extention.proto

```diff
@@ -4,4 +4,19 @@ option go_package = "github.com/grafana/grafana/pkg/services/authz/proto/v1";
 
 package authz.extention.v1;
 
-service AuthzExtentionService {}
+service AuthzExtentionService {
+  rpc List(ListRequest) returns (ListResponse);
+}
+
+message ListRequest {
```

Comment:
We probably need a verb here. Now we only can list objects user has `read` relation. But I expect a situation where we might need to list with a different relation, like `write` or `delete`.

Reply:
Sure!